atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1543,13 +1702,7 @@ public void addGarbageCollectionSentinelValues(TableReference tableRef, Set<Cell
     @Override
     public Multimap<Cell, Long> getAllTimestamps(TableReference tableRef, Set<Cell> cells, long ts) {
         AllTimestampsCollector collector = new AllTimestampsCollector();
-        try {
-            loadWithTs(tableRef, cells, ts, true, collector, deleteConsistency);
-        } catch (UnavailableException e) {
-            throw new InsufficientConsistencyException("Get all timestamps requires all Cassandra nodes to be up and available.", e);
-        } catch (Exception e) {
-            throw Throwables.throwUncheckedException(e);
-        }
+        loadWithTs(tableRef, cells, ts, true, collector, deleteConsistency);
```

Comment:
@schlosna From looking through the files, only this type of `UnavailableException` is used in cassandra. The `RuntimeException` version is only used in CQL from what I can tell. If it's meant to be the `RuntimeException` version, we should change it in a different PR. This PR is only for fixing checkstyle / findbugs so functionality shouldn't change if possible